extensions/swagger-ui/runtime/pom.xml

```diff
@@ -0,0 +1,128 @@
+                            <artifactId>quarkus-extension-processor</artifactId>
+                            <version>${project.version}</version>
+                        </path>
+                    </annotationProcessorPaths>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.webjars</groupId>
+                                    <artifactId>swagger-ui</artifactId>
+                                    <version>${swagger-ui.version}</version>
+                                </artifactItem>
+                            </artifactItems>
+                            <outputDirectory>${project.build.directory}/webjar-swagger-ui</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.google.code.maven-replacer-plugin</groupId>
+                <artifactId>replacer</artifactId>
+                <version>${replacer.version}</version>
+                <executions>
+                    <execution>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>replace</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <file>
+                        ${project.build.directory}/webjar-swagger-ui/META-INF/resources/webjars/swagger-ui/${swagger-ui.version}/index.html
+                    </file>
+                    <replacements>
+                        <replacement>
+                            <token>https://petstore.swagger.io/v2/swagger.json</token>
+                            <value>/openapi</value>
+                        </replacement>
+                    </replacements>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>3.1.0</version>
+                <executions>
+                    <execution>
+                        <id>copy-resources</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/classes/swagger-ui</outputDirectory>
```

Comment:
I'm not totally sold by the location of these files.

@stuartwdouglas do you have a preference on where we should put web resources that we don't want publicly accessible?

Reply:
Something I don't really like about this is that in JVM mode these are still copied into the uberjar or lib dir

Reply:
So I think an alternative approach would be to:
- Add the webjar as a dependency of the deployment module
- In dev mode as part of the build step extract this to a temp directory (and store the result in a static)
- Register the result as a special resource handler

This means that none of the swagger-ui ends up in the runtime artifact

Reply:
Does it make sense to create a `${cwd}/.quarkus` directory, copy the swagger-ui into and serve it?

Reply:
I think it would make more sense to just put it in a temp dir, end remove it with a shutdown hook

Reply:
@stuartwdouglas done.
I propose to close this conversation for readability sake and continue the discussion on this diff
https://github.com/quarkusio/quarkus/pull/1573/files#diff-3d22741acca0b68514ba03801f7818d3